src/services/address.service.ts

```diff
@@ -14,15 +14,41 @@ class AddressService {
         return;
       }
 
-      this.request(addressRequest)
-        .then((response) => {
-          resolve({
-            count: response.size(),
+      if (addressRequest.body.page != undefined) {
+        this.request(addressRequest)
+          .then((response: Array<Object>) => {
```

Comment:
Any reason to just not resolve the object without the array? Do you expect to return multiple responses?